dto/decision_dto.go

```diff
@@ -72,6 +72,12 @@ type PivotValue struct {
 	PivotId    null.String `json:"pivot_id"`
 }
 
+type DecisionData struct {
```

Comment:
This format is sufficient for the first graph described in figma, but not sufficient for the next one (histogram of decisions by score). IMO we should expose the more granular data on the backend, and then the frontend can perform an additional aggregation to display the right data ?